enterprise/backend/test/metabase_enterprise/sandbox/api/gtap_test.clj

```diff
@@ -194,26 +195,32 @@
 
 (deftest bulk-upsert-sandboxes-test
   (testing "PUT /api/permissions/graph"
-    (mt/with-temp* [Table                  [{table-id-1 :id}]
-                    Table                  [{table-id-2 :id}]
+    (mt/with-temp* [Table                  [{table-id-1 :id} {:db_id (mt/id), :schema "PUBLIC"}]
+                    Table                  [{table-id-2 :id} {:db_id (mt/id), :schema "PUBLIC"}]
                     PermissionsGroup       [{group-id :id}]
                     Card                   [{card-id-1 :id}]
                     Card                   [{card-id-2 :id}]]
       (premium-features-test/with-premium-features #{:sandboxes}
         (with-gtap-cleanup
           (testing "Test that we can create a new sandbox using the permission graph API"
-            (let [graph  (assoc (perms/data-perms-graph)
-                                :sandboxes [{:table_id             table-id-1
-                                             :group_id             group-id
-                                             :card_id              card-id-1
-                                             :attribute_remappings {"foo" 1}}])
+            (let [graph  (assoc (assoc-in (perms/data-perms-graph)
+                                          [:groups group-id (mt/id) :data :schemas "PUBLIC" table-id-1 :query]
+                                          :segmented)
+                                :sandboxes
+                                [{:table_id             table-id-1
+                                  :group_id             group-id
+                                  :card_id              card-id-1
+                                  :attribute_remappings {"foo" 1}}])
                   result (mt/user-http-request :crowberto :put 200 "permissions/graph" graph)]
-              (is (partial= [{:table_id table-id-1 :group_id group-id}]
-                            (:sandboxes result)))
-              (is (db/exists? GroupTableAccessPolicy
-                              :table_id table-id-1
-                              :group_id group-id
-                              :card_id  card-id-1))))
+              (is (schema=
```

Comment:
I think `=?` could simplify this check nicely.

Reply:
oh, didn't know that was merged, thanks!